api/src/main/java/net/kyori/adventure/text/Component.java

```diff
@@ -1358,6 +1440,36 @@ public interface Component extends ComponentBuilderApplicable, ComponentLike, Ex
     return translatable(requireNonNull(translatable, "translatable").translationKey(), color, decorations);
   }
 
+  /**
+   * Creates a translatable component with a translation key, optional fallback string, and optional color and decorations.
+   *
+   * @param key the translation key
+   * @param fallback the fallback string
+   * @param color the color
+   * @param decorations the decorations
+   * @return a translatable component
+   * @since 4.13.0
+   */
+  @Contract(value = "_, _, _, _ -> new", pure = true)
+  static @NotNull TranslatableComponent translatable(final @NotNull String key, final @Nullable String fallback, final @Nullable TextColor color, final TextDecoration@NotNull... decorations) {
+    return translatable(key, fallback, Style.style(color, decorations));
+  }
```

Comment:
Any new API should just really do a varargs/iterable of StyleBuilderApplicables - that way it can be done in just one method rather than the bunch that exist here